airbyte/_future_cdk/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+"""Module for future CDK components.
+
+Components here are planned to move to the CDK.
+
+TODO!: Add GitHub link here before merging.
+"""
+
+from __future__ import annotations
```

Comment:
great idea to move the base classes into future_cdk module! will be easy to move it out later.